.github/workflows/upload-release-assets.yml

```diff
@@ -15,10 +15,10 @@ jobs:
     strategy:
       matrix:
         include:
-          # - os: ubuntu-latest
-          #   npm_script: pack:linux
-          #   dist_folder: deb
-          #   extension: tar.gz #or deb, we do not know yet
+          - os: ubuntu-latest
+            npm_script: pack:linux
+            dist_folder: deb
+            extension: deb #or deb, we do not know yet
```

Comment:
@derberg I'm not sure what this step does? What's the significance of `extension` here? 

Also, I guess I have to add another command to package for `tar.gz`, right?(and create a new step with `extension: tar.gz`)

Reply:
it is used here `files: dist/${{ matrix.dist_folder }}/asyncapi.${{ matrix.extension }}` when pointing to binary file, and its extension

> Also, I guess I have to add another command to package for tar.gz, right?(and create a new step with extension: tar.gz)

yup cause I also guess you will not put dist into `deb` but something else